doc/wiki/Installation:-Option-List.md

```diff
@@ -0,0 +1,147 @@
+|:---:|:---:|:---:|---|---|---|
+| `--model`    | `HYDRO`, `ELBDM` | `HYDRO` | Physical models, where `ELBDM` is for &psi;DM | Must be set in any cases | <a name="--model"></a> `MODEL` |
+| `--gravity`  | `true`, `false`  | `false` | Enable [[gravity \| Gravity]] | Must enable `--fftw`; may need to set `FFTW2/3_PATH` in [[configuration file \| Installation:-Machine-Configuration-File#1-Library-paths]] | <a name="--gravity"></a> `GRAVITY` |
+| `--particle` | `true`, `false`  | `false` | Enable [[particles \| Particles]] | Must enable `--gravity` or `--tracer` | <a name="--particle"></a> `PARTICLE` |
+| `--grackle`  | `true`, `false`  | `false` | Enable [[GRACKLE \| Chemistry and Radiation]] | May need to set `GRACKLE_PATH` in [[configuration file \| Installation:-Machine-Configuration-File#1-Library-paths]]; only support `--eos=GAMMA/COSMIC_RAY`; does not support `--comoving` | <a name="--grackle"></a> `SUPPORT_GRACKLE` |
+| `--passive`  | &#8805; 0        | `0`     | Number of user-defined passive scalars | See [[here \| Adding-New-Simulations#v-add-problem-specific-grid-fields-and-particle-attributes]] for details; not supported for `--flu_scheme=RTVD` | <a name="--passive"></a> `NCOMP_PASSIVE_USER` |
+
+## Hydro Options
+-- see [[Hydro]] for the related runtime parameters and other settings
```

Comment:
Add "; must enable `--model=HYDRO`" at the end of this sentence. Modify other similar descriptions below each section too.